core/Translation/Translator.php

```diff
@@ -185,15 +188,36 @@ public function setCurrentLanguage($language)
     }
 
     /**
-     * @return string The default configured language.
+     * @return string The default language.
      */
     public function getDefaultLanguage()
```

Comment:
> Maybe we could move that code to a method in LanguagesManager and leave getDefaultLanguage as it was?

@tsteur Reasonable, done. In order to avoid making core dependent from plugins, it requires adding another hook: `Request.dispatch` to `LanguagesManager` with `initLanguage` in it - I guess, it is affordable, right?